tests/plugins/test_fetch_maven_artifacts.py

```diff
@@ -354,13 +360,23 @@ def mock_url_downloads(remote_files=None, overrides=None):
 
 
 @responses.activate  # noqa
-def test_fetch_maven_artifacts(tmpdir, docker_tasker):
+def test_fetch_maven_artifacts(tmpdir, docker_tasker, reactor_config_map):
     workflow = mock_workflow(tmpdir)
     mock_koji_session()
     mock_fetch_artifacts_by_nvr(str(tmpdir))
     mock_fetch_artifacts_by_url(str(tmpdir))
     mock_nvr_downloads()
     mock_url_downloads()
+
```

Comment:
Please refactor common code into a single function and call that function instead of repeating the common code in each test case.

Reply:
well it isn't that common as the map might be different like in the last case, but this way at least you see which map is it using,

and if I am to create function which will get (workflow, reactor_map)
I'll still have map setting in here an fnc call, which is just 1 less line of code

Reply:
I do think a helper function in this file would be helpful:

```python
def make_and_store_reactor_config_map():
   data = {
      'version': 1,
      'koji': {
        'hub_url': KOJI_HUB,
        'root_url': KOJI_ROOT,
        'auth': {}}}
    workflow.plugin_workspace[ReactorConfigPlugin.key] = {
        WORKSPACE_CONF_KEY: mocked_reactorconfig(data)
    }
```

Reply:
ok